docs/get_started/raspbian_setup.md

```diff
@@ -0,0 +1,76 @@
+# Installing MXNet on Raspbian
+MXNet currently supports the Debian based Raspbian operating system for Raspberry Pi Devices, offering a python API. 
+
+The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to this we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
+
+## Standard installation
+
+Installing MXNet is a two-step process:
+
+1. Build the shared library from the MXNet C++ source code.
+2. Install the supported language-specific packages for MXNet.
+
+### Build the Shared Library
+
+On Rasbian versions Wheezy and later, you need the following dependencies:
```

Comment:
I have not tested with earlier versions, but from my past experience with Pis you will likely need a significant amount of updates to packages etc and it is easier to just assume a common base distribution to work from.